e2e_tests/pytest.ini

```diff
@@ -13,6 +13,7 @@ markers =
     e2e_cpu_postgres: end to end CPU tests for testing basic database functionality
     e2e_cpu_cross_version: end to end CPU tests for testing basic cluster functionality with differing master/agent versions
     e2e_cpu_agent_connection_loss: end to end CPU tests for testing agent functionality on connection loss
+    e2e_cpu_streaming: end to end streaming updates tests
```

Comment:
why add a new kind of test?  Why not include this in the normal cpu test suite?

Reply:
I wonder what is the rule for splitting tests? I saw rbac, elastic being tested separately